deployment/helm/balloons/values.schema.json

```diff
@@ -114,11 +113,34 @@
                 }
             }
         },
-        "hostPort": {
-            "type": "integer"
-        },
         "podPriorityClassNodeCritical": {
             "type": "boolean"
+        },
+        "ports": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": [
+                    "name",
+                    "container",
+                    "host"
+                ],
+                "properties": {
+                    "name": {
+                        "type": "string"
+                    },
+                    "container": {
```

Comment:
```suggestion
                    "containerPort": {
```

Reply:
That's not necessary. I chose not to tautologically repeat 'Port' in the port object property names.